client/src/alumni/form.js

```diff
@@ -180,6 +180,7 @@ function AddAlumniForm(props) {
 
 
     function addToAlumniCollection(suc) {
+        successfulSubmitNotification();
```

Comment:
You did this in your other pr already right?

Reply:
Yes this was just making it fail. I think it had a bug or something. I put the logic on the server side in this one i think. I honestly don't remember